crawler/config_parser.py

```diff
@@ -7,8 +7,7 @@
 _config = None
 
 
-def parse_crawler_config(config_path='crawler.conf',
-                         options={}):
+def parse_crawler_config(config_path='crawler.conf'):
```

Comment:
Are we enforcing the location and name of this file instead of making it a command line argument?